### DISCUSSION TITLE
[ 功能建议 ]针对MySQL8添加字段优化建议

### DISCUSSION THREAD

**Discussion by jimsoft1000** [hhyo/Archery#1575] [Ideas]
### 目前遇到的问题/使用障碍
MySQL数据库增加字段使用ptosc工具，在MySQL8的版本，官方已经实现秒级新增字段，能否判断一下数据库的版本，如果是8以上的版本，新增字段时，可以直接执行，而不是使用ptosc工具

### 其他信息
如果有其他类似的产品功能或者图片信息，可在此提交

**Comment by LeoQuote** [comment]
mysql 执行是统一过goinception ,你看看 goinception 有无此类功能, 如果有的话, 直接改 goinception 就可以了

**Comment by jimsoft1000** [comment]
> 好的
